.copier-answers.image-template.yml

```diff
@@ -0,0 +1,15 @@
+# Changes here will be overwritten by Copier; do NOT edit manually
+_commit: v0.1.3
+_src_path: https://github.com/Tecnativa/image-template.git
+dockerhub_image: ""
```

Comment:
I think that for this particular image we can still use the docker hub, as it has no backwards incompatibilities and is supported. This is not like docker-duplicity, where we needed to change the publishing structure.

Reply:
Yes, I left this out for now as I think the current image on DockerHub is being built directly from the Dockerfile, as a responsibility of DockerHub, right? 
We'd need to disable that and then add it back here. Can you do that please?

Reply:
Done.